src/Container/Command/ConsumeCommandFactory.php

```diff
@@ -23,9 +23,9 @@ class ConsumeCommandFactory
     public function __invoke(ContainerInterface $container): ConsumeMessagesCommand
     {
         $config = $container->has('config') ? $container->get('config') : [];
-        $logger = $config['symfony']['messenger']['logger'] ?? null;
+        $logger = $config['framework']['messenger']['logger'] ?? null;
         $logger = $logger ? $container->get($logger) : null;
-        $receivers = $config['symfony']['messenger']['transports'] ?? [];
+        $receivers = $config['framework']['messenger']['transports'] ?? [];
```

Comment:
@gsteel 
If you go to the [documentation for Messenger on the Symfony site](https://symfony.com/doc/current/messenger.html#routing-messages-to-a-transport), you can see that they use 'framework'. 

I followed their lead, thinking it would cause less confusion for someone looking at the docs.

Reply:
Fair enough - I think that for the purposes of this lib, the configuration keys are obvious enough and changing them would be a big pain for users though, so I'm keen to keep `symfony` as the top-level key.